hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -2522,10 +2520,11 @@ protected OffPeakHours getOffPeakHours() {
 
   @Override
   public void onConfigurationChange(Configuration conf) {
-    this.conf = StoreUtils.createStoreConfiguration(conf, region.getTableDescriptor(),
+    Configuration storeConf = StoreUtils.createStoreConfiguration(conf, region.getTableDescriptor(),
       getColumnFamilyDescriptor());
-    this.storeEngine.compactionPolicy.setConf(conf);
-    this.offPeakHours = OffPeakHours.getInstance(conf);
+    this.conf = storeConf;
+    this.storeEngine.compactionPolicy.setConf(storeConf);
```

Comment:
OK, so the problem is here, the 'conf' here is the conf from the parameter, not 'this.conf'...

Reply:
Precisely!